provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -0,0 +1,89 @@
+# azure-iot-provisioning-http Requirements
+
+## Overview
+This module provides HTTP protocol support to communicate with the Azure device provisioning service
+
+```js
+'use strict';
+
+export class Http extends EventEmitter implements DeviceProvisioningTransport {
```

Comment:
>DeviceProvisioningTransport [](start = 50, length = 27)

would be interesting to see what this is.

Reply:
I will actually flush this out when I have 2 protocols.  Right now, I'm assuming it will be register+disconnect+maybe some of the strings like api-version.

---
In reply to: [140063985](https://github.com/Azure/azure-iot-sdk-node/pull/86#discussion_r140063985) [](ancestors = 140063985)

Reply:
sounds reasonable

---
In reply to: [140100915](https://github.com/Azure/azure-iot-sdk-node/pull/86#discussion_r140100915) [](ancestors = 140100915,140063985)